src/entities/sectionTitle/SectionTitle.css

```
@@ -0,0 +1,75 @@
+.section-title {
+    justify-content: center;
+    text-align: center;
+    margin: 21px auto 0;
+    width: 343px;
+    height: 47px;
+    background-color: #373F7B;
+    font-weight: 600;
+    font-size: 25px;
+    font-family: "Rubik";
+    color: #FFFFFF;
+    border-radius: 8px;
+    padding: 7px 0;
+}
+
+
+
+.section-title--2{
```

Если это `h2`, то в описании стиля лучше явно это выражать

```suggestion
.section-title--h2{
```

C `h3` и `h4` ниже аналогичная история